Formula/hivemind.rb

```diff
@@ -0,0 +1,23 @@
+class Hivemind < Formula
+  desc "The mind to rule processes of your development environment"
+  homepage "https://github.com/DarthSim/hivemind"
+  url "https://github.com/DarthSim/hivemind/archive/v1.0.tar.gz"
+  sha256 "7dde50f8f68214929f53c380ffc6311b39aad071a67bb6d94ebc3e641f78083a"
+
+  head "https://github.com/DarthSim/hivemind.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    ENV["GO15VENDOREXPERIMENT"] = "1"
```

Comment:
This is on by default with the version of `go` Homebrew ships, so you don't need to write it explicitly.